samples/unit-tests/chart/parallel-coordinates-updates/demo.js

```diff
@@ -167,49 +165,46 @@ QUnit.test('Take range series points into account while calculating yAxis extrem
     const firstAxisMin = chart.yAxis[0].dataMin,
         firstAxisMax = chart.yAxis[0].dataMax,
         thirdAxisMin = chart.yAxis[2].dataMin,
-        thirdAxisMax = chart.yAxis[2].dataMax;
+        thirdAxisMax = chart.yAxis[2].dataMax,
+        series = chart.series[0];
 
-    assert.deepEqual(23, firstAxisMin, '1st yAxis min should be 23.');
-    assert.deepEqual(25, firstAxisMax, '1st yAxis max should be 25.');
-
-    assert.deepEqual(1.5, thirdAxisMin, '3rd yAxis min should be 1.5.');
-    assert.deepEqual(4, thirdAxisMax, '3rd yAxis max should be 4.');
-});
+    assert.strictEqual(
+        series.points[0].low,
+        firstAxisMin,
+        '1st yAxis min should be 23.'
```

Comment:
Sorry, I forgot to point that you should also change `23` here too )(and below in other assertions). Something like this:
```The first yAxis' min should be equal to the first point's low value.```

Reply:
I hope that line that's length is equal to exactly 80 won't be a problem. 😄

Reply:
No, 80 is good ;)